config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -1749,6 +1749,20 @@
     </details>
   </checkerFrameworkError>
 
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java</fileName>
+    <specifier>not.interned</specifier>
+    <message>attempting to use a non-@Interned comparison operand</message>
+    <lineContent>if (node == currentCase.getLastChild().getLastChild()) {</lineContent>
+  </checkerFrameworkError>
+
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java</fileName>
+    <specifier>not.interned</specifier>
+    <message>attempting to use a non-@Interned comparison operand</message>
+    <lineContent>if (node == currentCase.getLastChild().getLastChild()) {</lineContent>
```

Comment:
Can we avoid this extension of suppression?

Reply:
give me time i have to learn about checker framework

Reply:
done